src/SFA.DAS.ProviderApprenticeshipsService.Web/DependencyResolution/DefaultRegistry.cs

```diff
@@ -212,21 +216,21 @@ private ProviderApprenticeshipsServiceConfiguration GetConfiguration(string envi
             return configurationService.Get<ProviderApprenticeshipsServiceConfiguration>();
         }
 
-        private ProviderRelationshipsReadStoreConfiguration GetProviderPermissionsReadStoreConfiguration(string environment, IConfigurationRepository configurationRepository)
-        {
-            var configurationService = new ConfigurationService(configurationRepository,
-                new ConfigurationOptions("SFA.DAS.ProviderRelationships.ReadStore", environment, "1.0"));
+        //private ProviderRelationshipsReadStoreConfiguration GetProviderPermissionsReadStoreConfiguration(string environment, IConfigurationRepository configurationRepository)
+        //{
+        //    var configurationService = new ConfigurationService(configurationRepository,
+        //        new ConfigurationOptions("SFA.DAS.ProviderRelationships.ReadStore", environment, "1.0"));
 
-            return configurationService.Get<ProviderRelationshipsReadStoreConfiguration>();
-        }
+        //    return configurationService.Get<ProviderRelationshipsReadStoreConfiguration>();
+        //}
 
-        private ProviderRelationshipsApiClientConfiguration GetProviderRelationshipsApiClientConfiguration(string environment, IConfigurationRepository configurationRepository)
-        {
-            var configurationService = new ConfigurationService(configurationRepository,
-                new ConfigurationOptions("SFA.DAS.ProviderRelationships.Api.Client", environment, "1.0"));
+        //private ProviderRelationshipsApiClientConfiguration GetProviderRelationshipsApiClientConfiguration(string environment, IConfigurationRepository configurationRepository)
+        //{
+        //    var configurationService = new ConfigurationService(configurationRepository,
+        //        new ConfigurationOptions("SFA.DAS.ProviderRelationships.Api.Client", environment, "1.0"));
 
-            return configurationService.Get<ProviderRelationshipsApiClientConfiguration>();
-        }
+        //    return configurationService.Get<ProviderRelationshipsApiClientConfiguration>();
+        //}
```

Comment:
is it worth leaving this uncommented but unused, such that it gets compile time checked at least and doesn't go stale when not in use?

Reply:
As discussed over Slack, there are pros and cons of leaving the code commented out vs having it uncommented.

If it's uncommented, then the code won't bitrot as breaking changes are made, but dead code will be included into the assembly, and developers might incorrectly assume the code is used, or notice it's not used and remove it (it would also add a maintenance burden).

We could possibly wrap the code in an 'if DEBUG' or similar, but it's probably best to consider the commented out code as documentation, and if/when someone needs to override the config, if the code compiles after being uncommented, then it is a bonus!

I also considered removing the commented-out code into a comment on the PR, but I thought that it would be unlikely be noticed.